scripts/biomedical/human1 data/README.md

```diff
@@ -0,0 +1,136 @@
+# Importing Human1 data

```

Comment:
What is the canonical name of the source or dataset? a google search for human1 doesn't show up anything.

also, do not have spaces in directory names (e.g., just scripts/biomedical/human1/). but ideally: scripts/biomedical/<source>/<dataset>/

Reply:
Modify to Human-GEM Human 1